libs/core/src/lib/feed-list-item/components/item/feed-list-item.component.html

```diff
@@ -47,16 +47,16 @@
 </ng-template>
 
 <ng-template #feedContent let-param>
+    <ng-container *ngTemplateOutlet="param"></ng-container>
     <span
-        class="lineclamp"
-        #lineClamp="fdLineClamp"
-        fd-lineclamp
-        [fdLineclampState]="isCollapsed"
-        [fdLineClampLines]="maxRows"
-        (lineCountUpdate)="checkLineCount($event)"
+        #truncate="fdTruncate"
+        fd-truncate
+        [fdTruncateState]="isCollapsed"
+        [fdTruncateChars]="maxChars"
+        [fdTruncateTargetText]="text"
+        (charCountUpdate)="checkCharCount($event)"
+        (update)="truncate.refreshTarget($event)"
```

Comment:
Seems like an event emitted from the directive and the method in that directive is called with the event as the parameter. Can we call the method from the directive itself and get rid of event emitting?

Reply:
Good point :) Updated